packages/firebase_data_connect/all_lint_rules.yaml

```diff
@@ -0,0 +1,183 @@
+# Copyright 2021 The Chromium Authors. All rights reserved.
```

Comment:
I think you don't need to duplicate this file. You are already including flutter_lints later so maybe it's better to rely on it.

Reply:
Removed